src/components/organisms/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import "./SparkleFairiesPopUp.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faAmbulance } from "@fortawesome/free-solid-svg-icons";
+
+interface PropsType {
+  children: React.ReactNode;
+}
+
+const SparkleFairiesPopUp: React.FunctionComponent<PropsType> = ({
+  children,
+}) => {
+  const [isLeftColumnExpanded, setIsLeftColumnExpanded] = useState(false);
+
+  return (
+    <div className="information-left-column-container">
+      <div
+        className={`left-column ${isLeftColumnExpanded ? "expanded" : ""}`}
+        onClick={() => setIsLeftColumnExpanded(!isLeftColumnExpanded)}
```

Comment:
bad practice to set state conditional on itself like this. Use `setIsLeftCo...(prevState => !prevState)` instead